pandas/plotting/_matplotlib/converter.py

```diff
@@ -872,14 +872,15 @@ def _annual_finder(vmin, vmax, freq):
 def get_finder(freq: BaseOffset):
     dtype_code = freq._period_dtype_code
```

Comment:
shouldnt this return FreqGroup too?

Reply:
there isnt a one-to-one mapping from period_dtype_code to FreqGroup

Reply:
k, maybe try to add a public method to freq which does this in the future